packages/web/src/pages/upload-page/fields/AccessAndSaleField.tsx

```diff
@@ -144,12 +144,14 @@ export const AccessAndSaleFormSchema = (trackLength: number) =>
           )
         })
       ),
-      [PREVIEW]: z.optional(z.nullable(z.number()))
+      [PREVIEW]: z.optional(z.nullable(z.number())),
+      [AVAILABILITY_TYPE]: z.nativeEnum(TrackAvailabilityType)
```

Comment:
https://github.com/colinhacks/zod/discussions/2125

Reply:
Yep that looks like what I've had to do before. :+1: